assets/keymaps/default-linux.json

```
@@ -576,6 +576,13 @@
       "pagedown": "editor::ContextMenuLast"
     }
   },
+  {
+    "context": "Editor && showing_multiple_signature_help && !showing_completions",
```

```suggestion
    "context": "Editor && can_scroll_signature_help",
```

Seems that we can simplify this by giving a more readable name and only adding this context when no completions/whatever else conditions.
WDYT?